README.markdown

```diff
@@ -12,11 +12,10 @@ working development environment for Rails up and running, including the database
 
 ## Ruby
 
-You will need Ruby 2.3.8 Verify that your path points to the correct version of Ruby:
+You will need the version of Ruby specified in `.ruby-version`. Verify that your path points to the correct version of Ruby:
 
-    devbox:usasearch
     $ ruby -v
-    ruby 2.3.8p459 (2018-10-18 revision 65136) [x86_64-darwin16]
+    ruby 2.5.5p157 (2019-03-15 revision 67260) [x86_64-darwin18]
 
```

Comment:
I just realized if we are not going to specify the version, which is good idea you put "the version of Ruby specified in '.ruby-version', " in line 15, you may want to do the same here.      ruby 2.5.5p157 (2019-03-15 revision 67260) [x86_64-darwin18] is too specific and users won't have that exact output when they run ruby -v.  They may have ruby 2.5.5p200 or something.  I would say " For example, you should see something similar to the following:"  or "The output should start with something like ruby 2.5.1"